website/docs/cloud-docs/no-code-provisioning/provisioning.mdx

```diff
@@ -34,7 +34,7 @@ To launch the no-code workflow:
 
 1. Select the no-code ready module to view its details, then click **Provision workspace**. The **Configure module inputs** page appears.
 
-     Terraform Cloud scans the module configuration for input variables and prompts for values for any variables without defaults or undefined in an existing global variable set. Terraform requires values for these variables to successfully complete runs in the workspace. Terraform Cloud performs type validation for the variable values if the module configuration specifies a type.
+     Terraform Cloud scans the module configuration for input variables and prompts for values for any variables without defaults or undefined in an existing [global or project-scoped variable set](/terraform/cloud-docs/workspaces/variables/managing-variables#variable-sets). Terraform requires values for these variables to successfully complete runs in the workspace. Terraform Cloud performs type validation for the variable values if the module configuration specifies a type.
```

Comment:
It's worth checking with Olivia to see if they will have this functionality in place.
Previously the variables come up before the project is selected, meaning there isn't a way to know if there are any project varset variables.

Reply:
> Terraform requires values for these variables to successfully complete runs in the workspace.

It seems like we're repeating ourselves in the bullet point below -- should we remove this sentence from this step and get rid of the "Optional" tag in the next step?

Reply:
Spoke with Olivia in the no-code sync. The removal of project-scoped variables from the list that ends up in the variable form is not part of the GA, so you can remove this change.